java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -644,7 +639,7 @@ protected int findHitPointType(Point2D hitPoint, boolean useRectangles, boolean
             Point2D p, minPoint = MathUtil.zeroPoint2D;
 
             double circleRadius = LayoutEditor.SIZE * layoutEditor.getTurnoutCircleSize();
-            double distance, minDistance = POSITIVE_INFINITY;
+            double distance, minDistance = Float.POSITIVE_INFINITY;
```

Comment:
There is a type mismatch here. The value is a double, but you are assigning a Float value. You should replace Float.POSITIVE_INFINITY with Double. POSITIVE_INFINITY

Reply:
You did not fix this one, you just marked the conversation resolved

Reply:
Sorry, missed that one… next commit should fix it.